packages/remote-config/lib/index.js

```diff
@@ -205,12 +265,21 @@ class FirebaseConfigModule extends FirebaseModule {
     return this._promiseWithConstants(this.native.setDefaultsFromResource(resourceName));
   }
 
+  setLogLevel() {
+    // eslint-disable-next-line no-console
+    console.warn('firebase.remoteConfig().setLogLevel() is not supported natively.');
+  }
+
   _updateFromConstants(constants) {
     this._lastFetchTime = constants.lastFetchTime;
     this._lastFetchStatus = constants.lastFetchStatus;
-    this._values = convertNativeConfigValues(constants.values);
-    this._isDeveloperModeEnabled = constants.isDeveloperModeEnabled;
-    this._minimumFetchInterval = constants.minimumFetchInterval;
+
+    this._settings = {
+      fetchTimeMillis: constants.fetchTimeout * 1000,
+      minimumFetchIntervalMillis: constants.minimumFetchInterval * 1000,
+    };
+
+    this._values = Object.freeze(constants.values);
```

Comment:
I did add an `Object.freeze` in the end because the `_values` are exposed on the object.